packs_source/weaponfx.db/AMR.RPjF4SvsMP3dolyX.js

```diff
@@ -1,4 +1,4 @@
-const {targetsMissed, targetTokens, sourceToken} = game.modules.get("lancer-weapon-fx").api.getMacroVariables(typeof messageId === "undefined" ? null : messageId, actor);
+const {targetsMissed, targetTokens, sourceToken} = game.modules.get("lancer-weapon-fx").api.getMacroVariables(this);
```

Comment:
No meaningful changes to macros. Switched to a more elegant method for passing data into the macro, which works in both v10, v11, and when running a macro manually.